momento-sdk/src/main/java/momento/sdk/config/StorageConfigurations.java

```diff
@@ -0,0 +1,123 @@
+package momento.sdk.config;
+
+import java.time.Duration;
+import momento.sdk.config.transport.GrpcConfiguration;
+import momento.sdk.config.transport.StaticTransportStrategy;
+import momento.sdk.config.transport.TransportStrategy;
+import momento.sdk.retry.DefaultRetryEligibilityStrategy;
+import momento.sdk.retry.FixedCountRetryStrategy;
+import momento.sdk.retry.RetryStrategy;
+
+/** Prebuilt {@link StorageConfiguration}s for different environments. */
+public class StorageConfigurations {
+
+  public static final int DEFAULT_MAX_RETRIES = 3;
+
+  /**
+   * Provides defaults suitable for a medium-to-high-latency dev environment. Permissive timeouts,
+   * retries, and relaxed latency and throughput targets.
+   */
+  public static class Laptop extends StorageConfiguration {
+
+    private Laptop(TransportStrategy transportStrategy, RetryStrategy retryStrategy) {
+      super(transportStrategy, retryStrategy);
+    }
+
+    /**
+     * Provides the latest recommended configuration for a dev environment.
+     *
+     * <p>This configuration may change in future releases to take advantage of improvements we
+     * identify for default configurations.
+     *
+     * @return the latest Laptop configuration
+     */
+    public static StorageConfiguration latest() {
+      return Laptop.v1();
+    }
+
+    /**
+     * Provides v1 configuration for a laptop environment. This configuration is guaranteed not to
+     * change in future releases of the Momento Java SDK.
+     *
+     * @return the v1 laptop configuration
+     */
+    public static StorageConfiguration v1() {
```

Comment:
let's maybe avoid putting any v1s into these for now, so we have some wiggle room to decide what we want default retries/timeouts to look like?

Reply:
so just rename this to `latest` and get rid of the `latest` above?

Reply:
Removed in [5fb1f90](https://github.com/momentohq/client-sdk-java/pull/358/commits/5fb1f90690c7d1764b36c95ede12a007bcf45c29)